sdk/eventhub/event-hubs/src/eventPosition.ts

```diff
@@ -226,3 +82,54 @@ export function isEarliestEventPosition(eventPosition: EventPosition): boolean {
 
   return false;
 }
+
+/**
+ * Gets the `EventPosition` corresponding to the location of the the first event present in the partition.
+ * Use this position to begin receiving from the first event that was enqueued in the partition
+ * which has not expired due to the retention policy.
+ */
+export const earliestEventPosition: EventPosition = {
+  offset: -1
+}
+
+
+/**
+ * Gets the `EventPosition` corresponding to the end of the partition, where no more events are currently enqueued.
+ * Use this position to begin receiving from the next event to be enqueued in the partion after an ``EventHubConsumer``
+ * is created with this position.
+ * @returns EventPosition
+ */
+export const latestEventPosition: EventPosition = {
+  offset: "@latest"
+}
+
+/**
+ * @ignore
+ * @internal
+ */
+export function validateEventPosition(position: EventPosition) {
+  
+  const offsetPresent = position.offset != undefined;
+  const sequenceNumberPresent = position.sequenceNumber != undefined;
+  const enqueuedOnPresent = position.enqueuedOn != undefined;
+
+  if ((offsetPresent && sequenceNumberPresent) || (offsetPresent && enqueuedOnPresent) || (enqueuedOnPresent && sequenceNumberPresent)) {
+    throw new TypeError(`${position} is an invalid value for EventPosition. Set only one of offset, sequenceNumber or enqueuedOn properties.`);
```

Comment:
I think this error will look like `[object Object] is an invalid value for EventPosition`?

Reply:
True, we will have to JSON.stringify() it. I was also considering not including it in the error message at all.  Thoughts?

Reply:
I don't think it needs to be in the error message personally.

Reply:
Removed